### ISSUE TITLE
"Groups and Scenes" devices 

### ISSUE THREAD

**Issue by mbaastrup** [jvmahon/homebridge-hubitat#42]
It seems like when I use a "Group activation device: switch" controlling some single switches, it is appearing in HomeHit as a light bulb and cannot be changed. 
Can this be fixed, or provide a path to where I can add/change these type of mappings.

Thanks for a very stable plugin, I have used it since you recommended me to go for the HE

**Comment by jvmahon** [comment]
I checked the code and from what I can tell, the device *should* appear as a switch if you choose the type "Group Switch", but when I tested it myself, I also got a bulb.  I'll look into this some more in the future.

However, I did include an override to the automatic testing. Read Section 5 of the Readme here: https://github.com/jvmahon/homebridge-hubitat

* As explained in that section and shown below, simply include the word "outlet" or "switch" in the Device Label for the group activation device, click "Save Device", and restart Homebridge. The device will then appear in HomeKit as a switch or outlet.
* 

![image](https://github.com/jvmahon/homebridge-hubitat/assets/15061942/448fcd05-3b78-42d5-b4ce-32e428ee1d90)
